README.md

```diff
@@ -123,7 +123,17 @@ turbolift foreach --repos repoFile2.txt sed 's/pattern2/replacement2/g'
 This creates a fork and clones all repositories listed in the `repos.txt` file into the `work` directory.
 You may wish to skip the fork and work on the upstream repository branch directly with the flag `--no-fork`.
 
-> NTLD: if one of the repositories in the list requires a fork to create a PR, omit the `--no-fork` flag and let all the repositories be forked. For now it's a all-or-nothing scenario.
+> NTLD: if one of the repositories in the list requires a fork to create a PR,
+> omit the `--no-fork` flag and let all the repositories be forked.
+> For now, it's an all-or-nothing scenario.
+
+Repositories are cloned with the git flag `--depth=1`.
+If you need the full commit history and tags, you can run the following command
+after running `turbolift clone`:
+
+```shell
+turbolift foreach git fetch --unshallow --tags
+```
```

Comment:
As this could trip people up, we'd like to see if this can be a little more visible using [this syntax](https://github.com/orgs/community/discussions/16925) - suggestion:

```suggestion
> **Note**
> Repositories are cloned with the git flag `--depth=1`.
> If you need the full commit history and tags, you can run the following command
> after running `turbolift clone`:
> 
> ```shell
> turbolift foreach git fetch --unshallow --tags
> ```
```

Reply:
I found out about that syntax recently, but I'm glad you suggested it here. I think that'll make this pop better for users. :grinning: